go/websockets/ws/message.go

```diff
@@ -0,0 +1,27 @@
+package ws
+
+// CADLITickMsg describes a CADLI Tick received from the CCData streamer
+type CADLITickMsg struct {
```

Comment:
Hey Will, should we move this struct out of the `ws` package as it's pretty ccdata specific, compared to the more generic `ws` package

Reply:
maybe in `products`

Reply:
Yeah, it could be.  I was thinking that this contains the WS message format though (including the json tags) rather than a struct representing the current state of e.g. CADLI BTC-USD which would be more product/instrument related.  What do you think?

Reply:
Thanks Will, makes sense.